README.md

```diff
@@ -8,38 +8,29 @@
   To learn more about Litmus Chaos Experiments & the Litmus Chaos Operator, 
   visit this link: [Litmus Docs](https://docs.litmuschaos.io/) 
 
-- The exporter is tied to a Chaosengine custom resource, which, 
-  in-turn is associated with a given application deployment.
+- The exporter is tied to a Chaos-operator deployment resource, which, 
+  in-turn is associated with all chaosengines in the cluster.
 
-- The exporter is typically deployed as a sidecar to the Litmus Experiment
-  Runner container in the engine-runner pod, but can be launched as a
+- The exporter is typically deployed as a sidecar to the Litmus Chaos Operator Deployment, but can be launched as a
   separate deployment as well. 
 
 - Two types of metrics are exposed: 
 
-  - Fixed: TotalExperimentCount, TotalPassedTests, TotalFailedTests which are derived 
+  - Fixed: ClusterTotalExperimentCount, ClusterTotalPassedCount, ClusterTotalFailedCount, EngineTotalExperimentCount, EnginePassedExperimentCount, EngineFailedExperimentCount, EngineWaitingExperimentCount  which are derived 
     from the ChaosEngine specification upfront
 
   - Dymanic: Individual Experiment Run Status. The list of experiments may 
     vary across ChaosEngines (or newer tests may be patched into it. 
     The exporter reports experiment status as per list in the chaosengine
 
 - The metrics are of type Gauge, w/ each of the status metrics mapped to a 
-  numeric value(not-executed:0, running:1, fail:2, pass:3)
-
-- The metrics carry the application_uuid as label (this has to be passed as ENV)
+  numeric value(not-executed:0, fail:1, running:2, pass:3)
 
 ## Steps to build & deploy: 
 
 ### Local Machine 
 
-- Set the application deployment (assuming a live K8s cluster w/ app) UUID as ENV (APP_UUID)
-
-- Set the ChaosEngine CR name as ENV (CHAOSENGINE) 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine.yaml
-
-- If the experiments are not executed, apply the ChaosResult CRs manually 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosresult.yaml
+- Set the application deployment (assuming a live K8s cluster w/ app and chaos-operator deployed), and give it ClusterRole authorisation similar to chaos-operator deployment 
```

Comment:
Launch the chaos-exporter deployment with a serviceaccount that has ClusterRole authorisation similar to chaos-operator deployment